openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js

```diff
@@ -742,6 +747,159 @@ describe('Glossary page should work properly', () => {
+    cy.get('[data-testid="saveButton"]').should('be.visible').click();
+    cy.get('.ant-modal-body').should('not.exist');
+
+    // Add reviewer
+    cy.get('[data-testid="add-reviewers"]')
+      .scrollIntoView()
+      .should('be.visible')
+      .click();
+
+    cy.get('.ant-modal-body').should('be.visible');
+    cy.get('[data-testid="searchbar"]')
+      .should('be.visible')
+      .type(GLOSSARY_TERM_WITH_DETAILS.reviewer);
+    verifyResponseStatusCode('@searchGlossaryTerm', 200);
+    cy.get('[data-testid="loader"]').should('be.visible');
+    cy.get('[data-testid="user-card-container"]').should('be.visible');
+    cy.get('[data-testid="checkboxAddUser"]').should('be.visible').click();
+    cy.get('[data-testid="save-button"]').should('be.visible').click();
+    cy.get('.ant-modal-body').should('not.exist');
+
+    interceptURL('POST', '/api/v1/glossaryTerms', 'createGlossaryTerms');
+    cy.get('[data-testid="save-glossary-term"]')
+      .scrollIntoView()
+      .should('be.visible')
+      .click();
+
+    cy.wait('@createGlossaryTerms').then(({ request }) => {
+      const synonym = GLOSSARY_TERM_WITH_DETAILS.synonyms.split(',');
+
+      expect(request.body).to.have.all.keys(
+        'description',
+        'displayName',
+        'mutuallyExclusive',
+        'name',
+        'glossary',
+        'references',
+        'relatedTerms',
+        'reviewers',
+        'tags',
+        'synonyms'
+      );
+      expect(request.body.displayName).equals(GLOSSARY_TERM_WITH_DETAILS.name);
+      expect(request.body.name).equals(GLOSSARY_TERM_WITH_DETAILS.name);
+      expect(request.body.description).equals(
+        GLOSSARY_TERM_WITH_DETAILS.description
+      );
+      expect(request.body.mutuallyExclusive).equals(false);
+      expect(request.body.glossary).equals(NEW_GLOSSARY.name);
+      expect(request.body.reviewers).has.length(2);
+      expect(request.body.references).has.length(1);
+      expect(request.body.references[0]).to.have.all.keys('name', 'endpoint');
+      expect(request.body.synonyms).has.length(synonym.length);
+      expect(request.body.synonyms).to.deep.equal(synonym);
+      expect(request.body.tags).has.length(1);
+      expect(request.body.tags[0]).to.deep.equal({
+        labelType: 'Manual',
+        state: 'Confirmed',
+        tagFQN: 'PersonalData.Personal',
+        source: 'Tag',
+      });
+    });
+  });
+
+  it('Verify details of created glossaryTerm', () => {
+    interceptURL(
+      'GET',
+      `/api/v1/glossaryTerms/name/*.${GLOSSARY_TERM_WITH_DETAILS.name}?fields=*`,
+      'getGlossaryTerm'
+    );
+    interceptURL(
+      'GET',
+      '/api/v1/permissions/glossaryTerm/*',
+      'waitForTermPermission'
+    );
+    visitGlossaryTermPage(GLOSSARY_TERM_WITH_DETAILS.name);
+
+    verifyResponseStatusCode('@getGlossaryTerm', 200);
+    verifyResponseStatusCode('@waitForTermPermission', 200);
+    cy.get('[data-testid="glossary-term"] > :nth-child(1)')
```

Comment:
See if we can improve this selector.

Reply:
Here i wanted to search within 1st part of the page to reduce the scope of the selection